tensorflow/lite/core/api/flatbuffer_conversions.cc

```diff
@@ -1059,7 +1059,7 @@ TfLiteStatus ParseDepthToSpace(const Operator* op,
   if (schema_params != nullptr) {
     params->block_size = schema_params->block_size();
   } else {
-    // TODO(b/157480169): We should either return kTfLiteError or fill in some
+    // old todo (b/157480169): We should either return kTfLiteError or fill in some
```

Comment:
we do want to keep this TODO as is. If it is triggering some failure in the CI then we'll need to fix the CI. Can you please change it back to TODO(b/): and then I'll take a look at what the CI error is.